module-apps/apps-common/ApplicationCommon.cpp

```diff
@@ -927,7 +927,8 @@ namespace app
 
     bool ApplicationCommon::isCurrentWindow(const std::string &windowName) const noexcept
     {
-        if (const auto &window = windowsStack().get(windowName); window != nullptr) {
+        auto topWindowName = windowsStack().get(topWindow);
+        if (const auto &window = windowsStack().get(*topWindowName); window != nullptr) {
             return window->getName() == windowName;
```

Comment:
```   
 bool ApplicationCommon::isCurrentWindow(const std::string &windowName) const noexcept
    {
        if (const auto &window = windowsStack().get(topWindow); window != std::nullopt) {
            return window == windowName;
        }
        LOG_ERROR("no window: %s", windowName.c_str());
        return false;
    }
```
get(...str) in this case returns std::nullopt if there is no window. Therefore: 1. you get name of the top window 2. you check if it's current window. 
I know it's my mistake in this place, but it would be best to change it for good :)

Reply:
sure :)